egs/fisher_swbd/s5/local/chain/run_tdnn_lstm_1a_svd.sh

```diff
@@ -0,0 +1,363 @@
+#!/bin/bash
+# same as run_tdnn_opgru_1a.sh, but replacing Norm-OPGRU with LSTMP.
```

Comment:
This comment needs to be updated to reflect what this is and what its baseline is.  Also I'm not crazy about the name.  Maybe it should be run_tdnn_lstm_1b.sh?  We can't add a new wart to the name every time we introduce a new technique.  
Also please make sure the command in the comment reflects how you obtained the WERs.  Right now you have put some WERs in manually but it's not clear what directories they come from or how they were obtained.

Reply:
This PR is deprecated and now shifted to new PR - https://github.com/kaldi-asr/kaldi/pull/3272.